src/main/java/ArchipelagoMW/LocationTracker.java

```diff
@@ -121,7 +130,21 @@ static public String sendRelic() {
     /**
      * sends the next boss relic location to AP
      */
-    static public String sendBossRelic() {
+    static public String sendBossRelic(int act) {
+        ArrayList<Long> bossRelicLocations;
+        logger.info("going to send boss relic from act " + act);
+        switch (act){
+            case 1:
+                bossRelicLocations = bossRelicLocationsAct1;
+                break;
+            case 2:
+                bossRelicLocations = bossRelicLocationsAct2;
+                break;
+            case 3:
+            default:
+                bossRelicLocations = bossRelicLocationsAct3;
+                break;
+        }
```

Comment:
I'm not sure why this was split, in lists the order is preserved so you could have just sent `bossRelicLocations.get(0)` for act 1 `bossRelicLocations.get(1)` for act 2, etcetera.. this introduces lists of exactly one element and that seems very unnecessary. Then just delete the removal down below.

Reply:
quite a few things will have to be changed in that case, the way I made it allows for minimal changes to the original code. If I start using `bossRelicLocations.get(0)` instead of `bossRelicLocations.remove(0)` we will have to start using a counter for each of the lists and also scout all locations from the start instead after every check.
But sure, it's doable.

Reply:
Since we don't display the boss relic item to the player it does not need to be scouted really. The locations are only scouted to show the player who gets sent what. Its probably also worth it to just change it to scout everything upon connect. So we can avoid having to scout as we go. (not sure why I didn't do this to begin with.)

Reply:
Since I intend to add distinction between acts for items, scouting from the start will now be very much needed anyway for basically all locations. Shame I already started working on that since I was using the same template as I have already proposed with using different lists for each act :P I'm guessing I shouldn't start expanding on something that hasn't been approved yet.

Reply:
I don't like splitting the items between acts. And would instead prefer to use the checkpoint resuming where you left off with caps when using the portals.

Reply:
This will remain as an option in yaml. I will default it to the "checkpoint"  system you mentioned